src/applications/disability-calculator/components/DisabilityRatingCalculator.jsx

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable no-console */
+import React from 'react';
+import { calculateRating, roundRating } from '../utils/helpers';
+import '../sass/disability-calculator.scss';
+import { CalculatedDisabilityRating } from './CalculatedDisabilityRating';
+import { RatingRow } from './RatingRow';
+
+export default class DisabilityRatingCalculator extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      ratings: [
+        {
+          rating: 0,
+          description: '',
+          canDelete: false,
+        },
+        {
+          rating: 0,
+          description: '',
+          canDelete: false,
+        },
+        {
+          rating: 0,
+          description: '',
+          canDelete: true,
+        },
+      ],
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleRatingCalculateChange = this.handleRatingCalculateChange.bind(
+      this,
+    );
+    this.handleAddRating = this.handleAddRating.bind(this);
+    this.handleRemoveRating = this.handleRemoveRating.bind(this);
+    this.ratingRef = React.createRef();
+  }
+
+  handleClick = () => {
+    this.child.ratingInput.focus();
+  };
+
+  handleChange = (e, idx) => {
+    const curRatings = this.state.ratings;
+    curRatings[idx][e.target.name] =
+      // eslint-disable-next-line radix
+      e.target.name === 'rating' ? parseInt(e.target.value) : e.target.value;
+    this.setState({ ratings: curRatings }, () => {
+      console.log(this.state);
+    });
+  };
```

Comment:
I'm not sure if this `handleChange ` works or not, but this feels very frail. I think we can address this later though after getting the add/delete/clear handlers cleaned up.